{{cookiecutter.project_slug}}/Pipfile

```diff
@@ -28,5 +28,5 @@ drf-yasg = "~=1.20.0"
 fcm-django = "~=0.3.10"
 pyyaml = "~=5.4.1"
 pillow = "~=8.2.0"
-pytest = "==6.2.3"
+pytest = "==6.2.4"
```

Comment:
I believe these are from dependabot here https://github.com/crowdbotics/django-scaffold/pull/219/commits/84ac43adc211b8b12046050150ef7c9835495fec. We haven't had a release since May 6th so all the changes have been accumulating in develop.

Reply:
`develop` is set as the main branch for this repo so I guess dependabot sends all the PRs to `develop` instead of `master`